src/App.js

```diff
@@ -41,7 +41,9 @@ export default function App() {
 
   return (
     <div className="App">
-      <Router>
+      <Nav />
+      <Router />
```

Comment:
Great Way of Abstracting Router into separate component\
to make *App.js* look clean. :thumbsup:

Reply:
Thanks,
This one of the suggestions in Suyash's Review tho